frontend/components/PartcadViewerShell.tsx

```
@@ -143,56 +235,62 @@ export function PartcadViewerShell({ src }: Props) {
+          {jobState.message}
+        </div>
       </div>
     );
   }
 
   if (jobState.status !== 'ready') {
     return (
-      <div role="status" className="viewer-loading">
-        Preparing 3D model…
+      <div className="viewer-shell">
+        <output role="status" aria-live="polite" className="viewer-loading">
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)